src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -46,8 +46,8 @@ error.bad.cflag=\
 error.bad.uflag=\
         'u' flag requires manifest, 'e' flag or input files to be specified!
 error.bad.eflag=\
-	'e' flag and manifest with the 'Main-Class' attribute cannot be specified \n\
```

Comment:
Here were two lines that used tab instead of space; I converted them to 8 spaces.